app/src/NativeBase/Containers/ProfileContainer.tsx

```diff
@@ -13,20 +44,72 @@ import React from 'react'
 const ProfileContainer = () => {
   return (
     <VStack
-      paddingTop="4"
-      paddingBottom="2"
-      alignItems="center"
-      space="4"
-      backgroundColor="secondaryGrey.60"
+      padding={2}
+      backgroundColor="bg.100"
       _dark={{ backgroundColor: 'bgDarkMode.100' }}
     >
-      <Center
-        backgroundColor="bg.100"
-        _dark={{ backgroundColor: 'bgDarkMode.100' }}
-        width="100%"
-      >
-        <Heading size="sm">My Profile</Heading>
+      <Center>
```

Comment:
If the content of this screen ends up being taller than the available space, it doesn't seem to scroll -- content is just cut off e.g. on my phone (iPhone SE 2020) I can't see beyond My Projects and My Experience at the moment.

Reply:
Thanks for noticing Joe, it fitted on my screen so didn't realise that. The scrollview should wrap the whole content, not just the wee list. Have fixed :)

Reply:
Great, thanks -- that's working now for me 👍